ingest/src/main/java/com/microsoft/azure/kusto/ingest/QueuedIngestClientImpl.java

```diff
@@ -1,293 +1,293 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-package com.microsoft.azure.kusto.ingest;
-
-import com.azure.data.tables.models.TableEntity;
-import com.azure.storage.blob.models.BlobStorageException;
-import com.azure.storage.common.policy.RequestRetryOptions;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.microsoft.azure.kusto.data.*;
-import com.microsoft.azure.kusto.data.auth.ConnectionStringBuilder;
-import com.microsoft.azure.kusto.ingest.exceptions.IngestionClientException;
-import com.microsoft.azure.kusto.ingest.exceptions.IngestionServiceException;
-import com.microsoft.azure.kusto.ingest.result.IngestionResult;
-import com.microsoft.azure.kusto.ingest.result.IngestionStatus;
-import com.microsoft.azure.kusto.ingest.result.IngestionStatusInTableDescription;
```

Comment:
Are there any changes in this file?

Reply:
Ah, seems Line break type changed from LF to CRLF